.github/workflows/auto-comment.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
             WeChat  Group:
 
-            ![image](https://user-images.githubusercontent.com/7869972/176336986-d6b9be8f-d1d3-45f1-aa45-8e6adf5dd244.png)
+            ![image](https://github.com/apache/incubator-linkis-website/blob/master/static/Images/wedatasphere_contact_01.png)
```

Comment:
the image url  can not work fire 
maybe we should use as `https://linkis.apache.org/zh-CN/home/after_linkis_zh.png`

Reply:
Done